src/modelmanager.cpp

```diff
@@ -530,28 +530,72 @@ Status ModelManager::tryReloadGatedModelConfigs(std::vector<ModelConfig>& gatedM
     return firstErrorStatus;
 }
 
+class LoudFileInfoReporter {
+    const std::string filename;
+    std::stringstream ss;
+
+public:
+    LoudFileInfoReporter(const std::string& name) :
+        filename(name) {
+        if (filename == "") {
+            return;
+        }
+        struct stat statTime;
+
+        if (stat(filename.c_str(), &statTime) != 0) {
+            SPDLOG_ERROR("Failed to debug-read fileconfig");
+            return;
```

Comment:
What's the point of checking stat before opening the file?

Reply:
To leave time stamp as debug info